coral/vite.config.ts

```diff
@@ -147,6 +147,13 @@ export default defineConfig(({ mode }) => {
         FEATURE_FLAG_EDIT_CONNECTOR: ["development", "remote-api", "local-api"]
           .includes(mode)
           .toString(),
+        FEATURE_FLAG_CONFIGURATION_LINKS: [
```

Comment:
Small change I would suggest: Calling it `FEATURE_FLAG_CONFIGURATION`, then it can be used for the configuration pages, too. That way we only have to remove one flag when the feature is ready.

Reply:
Renamed [bb32538](https://github.com/Aiven-Open/klaw/pull/1713/commits/bb32538926b4f3363f0b7b1b590477dc2a1dcb90)